.ci/publishHelmChart.sh

```diff
@@ -0,0 +1,10 @@
+PACKAGE_VERSION=$(jq .version -r package.json)
+
+helm package --app-version $PACKAGE_VERSION --version $PACKAGE_VERSION ./helmChart
+
+path="./connector-helm-chart-$PACKAGE_VERSION.tgz"
+registry="oci://ghcr.io/nmshd"
```

Comment:
Kleinigkeit: Warum sind die Variablen klein? Überall anders haben wir sie glaub ich in Caps geschrieben.